### ISSUE TITLE
Show lyrics from tags

### ISSUE THREAD

**Issue by abba23** [dmo60/lLyrics#67]
It would be nice to have the option to use the lyrics from the music files' tags and only search remote sources as a fallback.

**Comment by ratijas** [comment]
I wonder, is it so hard or what? AFAIK only iTunes supports this out of the box in concise usable way.

**Comment by tralph3** [comment]
Oh my god... Years have passed and this has not been implemented. I really need something like this, I spent a shit ton of time putting lyrics on all of my mp3 files to get them replaced by shitty online results.

**Comment by ratijas** [comment]
Wellll, years are passing by, but nobody's developing this anymore.  Will you be the one?

**Comment by tralph3** [comment]
@ratijas Actually, yes, I was the one. I modified a plugin to fetch synced lyrics from tags, but it's for other player called Quod Libet. You can check it out in my "issues" tab. I'm actually quite proud.

_Sent from my P8 Lite using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by alpemwarrior** [comment]
I'm working on it... For now I can read from .lrc or .lyric files inside the same directory and use them as a source. The problem with tags is that each file stores the information differently and I can't seem to find a library that automatically extracts the lyrics, so I'll have to do it manually.

**Comment by ratijas** [comment]
AFAIK, iTunes does it somehow — manages lyrics for any file type out there. Maybe a little bit research wouldn't hurt?

I'd say, iTunes is the _de-facto_ golden standard when it comes to things like these. Not that I'm using it anymore, though~

**Comment by alpemwarrior** [comment]
Yep. I was hoping that there would be some library that could automagically extract the lyrics from any file, but no such luck. For now i'll focus on getting id3 and vorbis tags working and then i'll see.

**Comment by alpemwarrior** [comment]
Ok, mostly done. Just a stupid question, how can I add dependencies to the installation script (I need the mutagen library to read the metadata of the songs)?

**Comment by alpemwarrior** [comment]
Oh well, it seems like you'll have to install it manually. I'll update the readme and open the pull request then.

**Comment by alpemwarrior** [comment]
#79 Done
